modules/address/components/Jazzicon.tsx

```diff
@@ -63,8 +63,8 @@ export default function Jazzicon({ address, size }: Props) {
       const angle = Math.PI * 2 * firstRot;
       const velocity = (size / shapeCount) * generator.random() + (index * size) / shapeCount;
 
-      const tx = Math.cos(angle) * velocity;
-      const ty = Math.sin(angle) * velocity;
+      const tx = (Math.cos(angle) * velocity).toFixed(1);
+      const ty = (Math.sin(angle) * velocity).toFixed(1);
```

Comment:
That's used to generate the random profile pic for addresses 
![image](https://user-images.githubusercontent.com/22449631/236270251-22cf0320-b933-4f8b-b889-0e4a634aa477.png)

That specific change is a fix for one of the React hydration errors, the `tx` and `ty` values seemed to mismatch between server and client on the last couple of decimals (around 14th to 16th decimal), probably because of the pseudo-random generation on this component